clients/pkg/promtail/targets/file/filetarget_test.go

```diff
@@ -17,6 +21,43 @@ import (
 	"github.com/grafana/loki/clients/pkg/promtail/targets/testutils"
 )
 
+func createWatchers(ctx context.Context, path string) (chan fsnotify.Event, chan fileTargetEvent, error) {
```

Comment:
Now that `FileTaget` is not depending on any files anymore I wonder if this actually required. There's no need to write actual files to test `FileTarget`. This could be tested with the `FileTargetManager`. However, such a refactoring is a big ask.

Reply:
> There's no need to write actual files to test FileTarget. 
I think so too and I think such a change is a little bit large as you said 😭 
If you are OK, can I refactor this test in another PR?

Reply:
> If you are OK, can I refactor this test in another PR?

Of course. Be my guest :slightly_smiling_face: